assignments/closure.js

```diff
@@ -1,11 +1,28 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function foo() {
```

Comment:
add console logs to check your code and follow the usage guidelines in the code to check for funcitonality